mne/decoding/base.py

```diff
@@ -746,3 +758,86 @@ def _make_scorer(scoring):
         return get_scorer(scoring)
     else:
         return make_scorer(scoring)
+
+
+def _get_inverse_funcs(estimator, terminal=True):
+    """Retrieve the inverse functions of an pipeline or an estimator."""
+    inverse_func = [False]
+    if hasattr(estimator, 'steps'):
+        # if pipeline, retrieve all steps by nesting
+        inverse_func = list()
+        for _, est in estimator.steps:
+            inverse_func.extend(_get_inverse_funcs(est, terminal=False))
+    elif hasattr(estimator, 'inverse_transform'):
+        # if not pipeline attempt to retrieve inverse function
+        inverse_func = [estimator.inverse_transform]
+
+    # If terminal node, check that that the last estimator is a classifier,
+    # and remove it from the transformers.
+    if terminal:
+        last_is_estimator = inverse_func[-1] is False
+        all_invertible = not(False in inverse_func[:-1])
+        if last_is_estimator and all_invertible:
+            # keep all inverse transformation and remove last estimation
+            inverse_func = inverse_func[:-1]
+        else:
+            inverse_func = list()
+
+    return inverse_func
+
+
+def get_coef(estimator, attr='filters_', inverse_transform=False):
```

Comment:
wait didn't I write basically this function a little while back? Did that not get merged? Maybe it was a part of the original `EncodingModel` PR debacle?